buildSrc/reaper/build.gradle

```diff
@@ -20,3 +20,11 @@ jar {
     attributes 'Main-Class': 'org.opensearch.gradle.reaper.Reaper'
   }
 }
+
+repositories {
```

Comment:
Why do we need these changes here?

Reply:
To get around the issue I reported above:

```
Could not determine the dependencies of task ':build-tools:reaper:missingJavadoc'.
 Could not resolve all task dependencies for configuration ':build-tools:reaper:missingdoclet'.
    Cannot resolve external dependency org.jboss.spec.javax.annotation:jboss-annotations-api_1.2_spec:1.0.2.Final because no repositories are defined.
     Required by:
         project :build-tools:reaper > project :missing-doclet
```

Reply:
Oh sorry, I missed that